packages/twenty-emails/tsup.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig([
+  {
+    entry: { index: './tsup.index.tsx' },
+    format: ['cjs', 'esm'],
+    dts: true,
+    outDir: '../twenty-server/src/emails/generated',
```

Comment:
@martmull Sounds really great !
Not sure if it's a good way to send the bundled files to the server.
Can we instead use `twenty-emails` as a node package instead ?